test/unit/mdc-menu-surface/mdc-menu-surface.test.js

```diff
@@ -45,12 +45,28 @@ function getFixture(open, fixedPosition = false) {
   `;
 }
 
-function setupTest(open = false, fixedPosition = false) {
+function getAnchorFixture(menuFixture) {
+  const anchorEl = bel`
+    <div class="mdc-menu-surface--anchor">
+      <button class="mdc-button">Open</button>
+    </div>
+  `;
+  anchorEl.appendChild(menuFixture);
+  return anchorEl;
+}
+
+function setupTest(open = false, fixedPosition = false, withAnchor = false) {
```

Comment:
Can we make these arguments an object so we don't have a nebulous call to `setupTest(false, false, true)`?

Reply:
If I modify about 24 other unrelated lines of the file in this PR, technically yes